content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/saml-migration.md

```diff
@@ -0,0 +1,141 @@
+
+To update your existing SAML configuration with Okta:
+
+1. In Okta, go to your SAML application for Cobalt. On the **General** tab, under **SAML Settings**, select **Edit**, then select **Next**.
+    - **Single sign-on URL**: Enter **ACS URL** from Cobalt.
+    - Delete the **Default Relay State** value.<br><br>
+    ![Update SAML settings in the Cobalt SAML app in Okta](/deepdive/Cobalt-Okta-SAML-app-update-1.png "Update SAML settings in the Cobalt SAML app in Okta")
+1. In Okta, go the **Sign On** tab, and select **Edit**. Under **SAML 2.0**, select **More details**.
+    - Download the **Signing Certificate**.<br><br>
+    ![Download the Signing Certificate in the Cobalt SAML app in Okta](/deepdive/Cobalt-Okta-SAML-app-update-2.png "Download the Signing Certificate in the Cobalt SAML app in Okta")
+1. In Cobalt, go to **Settings** > **Identity & Access**. Under **Configure SAML**, select **Configure**.
+    - **IdP Certificate**: Enter the **Signing Certificate** from Okta.<br><br>
+    ![Update SAML settings for Okta in Cobalt](/deepdive/Cobalt-Okta-SAML-app-update-3.png "Update SAML settings for Okta in Cobalt")
+
+If you want to set up a new application, see the [general workflow](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#general-configuration-workflow) and configuration parameters for [Okta](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#okta).
+
+### Azure AD
+
+To update your existing SAML configuration with Azure AD:
+
+1. In Azure AD, go to your SAML application for Cobalt.
+    - **Reply URL (Assertion Consumer Service URL)**: Enter **ACS URL** from Cobalt.
+    - Delete the **Relay State** value.
+1. In Cobalt, go to **Settings** > **Identity & Access**. Under **Configure SAML**, select **Configure**.
+    - **IdP Certificate**: Enter **Certificate (Base64)** from Azure AD.
+
+If you want to set up a new application, follow this [instruction](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#azure-ad).
+
+### Duo
+
+To update your existing SAML configuration with Duo:
+
+1. In Duo, go to your SAML application for Cobalt.
+    - **Assertion Consumer Service (ACS) URL**: Enter **ACS URL** from Cobalt.
+    - Delete the **Default Relay State** value.
+1. In Cobalt, go to **Settings** > **Identity & Access**. Under **Configure SAML**, select **Configure**.
+    - **IdP Certificate**: Enter **Certificate** from Duo.
+
+If you want to set up a new application, see the [general workflow](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#general-configuration-workflow) and configuration parameters for [Duo](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#duo).
+
+### Google
```

Comment:
I think things are a little reverse for Google. we should probably just say to create a new app. IDK that you can go back to step 1 to get the certificate again? https://support.google.com/a/answer/6087519?hl=en&sjid=15243547827955905471-NA